packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -4,15 +4,15 @@
 %global foreman_min_version 1.18.0
 %global plugin_name katello
 %global gem_name katello
-%global prever .rc1
+%global prever .rc2
 %global mainver 3.7.0
 %global release 1
 
 Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Content and Subscription Management plugin for Foreman
 
-Version: %{mainver}%{?prever}
-Release: %{?prever:0.}%{release}%{?prever}%{?dist}
+Version: %{mainver}
+Release: %{release}%{prever}%{?dist}
```

Comment:
I had to change this b/c obal was telling me there wasn't a corresponding changelog entry for some mangled-looking version

Reply:
This line will need the prever -- https://github.com/theforeman/foreman-packaging/pull/2756/files#diff-f7d8204599e2f8844c3a647239ed441bR186

Reply:
gets me every time